demo/app.py

```diff
@@ -51,6 +51,26 @@
             CONTRACT_TYPES[0],
             id="ctr-type",
         ),
+        html.Br(),
+        dcc.RadioItems(
+            options=[
+                {'label': 'Call', 'value': 'Call'},
+                {'label': 'Put', 'value': 'Put'}
+            ],
+            value='Call',
+            id='ctr-option-type',
+            inline=True,
+            labelStyle={'margin-center': '20px'}  # Add spacing between radio buttons
+        ),
+        html.Br(),
+        dcc.Slider(
```

Comment:
Add a label to the left 'strike' so one knows this is strike.
Same for Cap/Floor.